bulksendemail.js

```diff
@@ -0,0 +1,258 @@
+/*
+ * @package     Plg_System_Tjlms
+ * @subpackage  Plg_System_Tjlms
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2018 Techjoomla. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
```

Comment:
@komalm  Create a UI file which should something like following 


// This constructor internally will call the init method which will take care of 
1. add checkboxes if not exist on a page
2. add send email button on the toolbar
`var email = new TjEmail(id, class);`


// This will take care of  crafting popup html
`email.prepare(); `


 // This will internally call ajax which should be written in the service file. 
1. Validate
2. Call service for sending email 
`email.send();`